openquake/calculators/base.py

```diff
@@ -436,7 +436,7 @@ def pre_execute(self):
             # there is a precalculator
             precalc_id = self.oqparam.hazard_calculation_id
             self.precalc = (self.compute_previous() if precalc_id is None
-                            else self.read_previous(precalc_id))
+                            else self.read_previous(int(precalc_id)))
```

Comment:
Ok, these were the first place where the id of the old calculation cannot be None and then can be casted to an int